webapp/src/components/NewsForm/NewsForm.stories.ts

```diff
@@ -0,0 +1,32 @@
+import NewsForm from './NewsForm.vue';
+
+export default {
+  title: 'components/NewsForm',
+  component: NewsForm,
+  argTypes: {
+    onCreate: {
+      action: 'Create Item',
+    },
+  },
+  /* parameters: {
+    a11y: {
+      // optional selector which element to inspect
+      element: '#root',
+      // axe-core configurationOptions (https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#parameters-1)
+      config: {},
+      // axe-core optionsParameter (https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#options-parameter)
+      options: {},
+      // optional flag to prevent the automatic check
+      manual: true,
+    },
+  }, */
```

Comment:
Here is some code commented out, better remove it if not needed before merging to master